region/norfair/crocomire/Post Crocomire Farming Room.json

```diff
@@ -627,6 +654,31 @@
         }
       }
     },
+    {
+      "link": [4, 4],
+      "name": "Leave With Runway - Frozen Gamet",
+      "requires": [
+        {"or": [
+          "h_canTrickyFrozenEnemyRunway",
+          {"and": [
+            "h_canFrozenEnemyRunway",
+            "Morph"
+          ]}
+        ]}
+      ],
+      "exitCondition": {
+        "leaveWithRunway": {
+          "length": 5,
+          "openEnd": 0
+        }
+      },
+      "note": [
+        "Lure the Geemers up; it may be helpful to freeze them on the ascent to prevent them from separating and moving horizontally.",
```

Comment:
```suggestion
        "Lure the Gamets up; it may be helpful to freeze them on the ascent to prevent them from separating and moving horizontally.",
```